backend/src/oauth/provider/discord.provider.ts

```diff
@@ -81,7 +81,11 @@ export class DiscordProvider implements OAuthProvider<DiscordToken> {
     if (guild) {
       await this.checkLimitedGuild(token, guild);
     }
-
+    const limitedusers = this.config.get("oauth.discord-limitedUsers");
```

Comment:
Sorry for being so picky but could you please rename the variable to `limitedUsers`?
```suggestion
    const limitedUsers = this.config.get("oauth.discord-limitedUsers");
```